src/MumbleProtocol.h

```diff
@@ -89,8 +89,9 @@ namespace Protocol {
 	};
 
 	namespace ReservedTargetIDs {
-		constexpr unsigned int REGULAR_SPEECH  = 0;
-		constexpr unsigned int SERVER_LOOPBACK = 31;
+		constexpr unsigned int REGULAR_SPEECH          = 0;
+		constexpr unsigned int SERVER_LOOPBACK_REGULAR = 30;
```

Comment:
This means that old clients on new servers will now be heard if they perform a "server" loopback test, correct?

Reply:
No - the previously existing special target for loopback is `31`, which retains its original meaning - as it should, imo.

Reply:
Ah damn I got confused by the order here ok